src/frontend/src/flows/compatibilityNotice.ts

```diff
@@ -1,26 +1,31 @@
-import { compatibilityChart } from "$src/components/compatibilityChart";
 import { mainWindow } from "$src/components/mainWindow";
 import { warnBox } from "$src/components/warnBox";
 import { html, render } from "lit-html";
 
 // Taken from: https://caniuse.com/?search=PublicKeyCredential
-const pageContent = (reason: string) => html`
-  <h1 class="t-title t-title--main" id="compatibilityNotice">
-    Internet Identity does not support your browser
-  </h1>
-  ${warnBox({
-    title: "Reason",
-    message: reason,
+const pageContent = () =>
+  warnBox({
+    title: "Browser Not Supported",
+    message: html`
+      <p class="t-paragraph">
+        Internet Identity is not supported by your browser.
+      </p>
+      <p class="t-paragraph">
+        This device does not offer
+        <a href="https://webauthn.io" rel="noopener noreferrer" class="t-link"
```

Comment:
I'm not sure we want to redirect users to webauthn.io. We could also use https://fidoalliance.org/passkeys/ since we're referring to passkeys everywhere on the front-end. This is also the official site of the industry association behind the WebAuthn standard.

Reply:
Sure thing! should I rename it to `Passkeys` then?

Reply:
Yes, I would recommend that. We can also ask @marydwyer what wording she prefers.